src/pages/GovPages/VoteStatus.ts

```diff
@@ -0,0 +1,11 @@
+export const InProgress = 'IN_PROGRESS'
+export const Passed = 'PASSED'
+export const Failed = 'FAILED'
+
+export enum VoteStatus {
+  IN_PROGRESS = 'purple3',
+  PASSED = 'green2',
+  FAILED = 'red1'
+}
+
+export type VoteStatusType = typeof InProgress | typeof Passed | typeof Failed
```

Comment:
Wouldn't this simply resolve to `export type VoteStatusType = string`?
I think it's a bit weird to have the enum do the role of a map above, but we can leave it like that if you want.

Reply:
![image](https://user-images.githubusercontent.com/54032883/107374742-c6578900-6a9c-11eb-9f4c-1191220265b8.png)